packages/trader/src/AppV2/Components/DraggableList/draggable-list.tsx

```diff
@@ -56,19 +62,37 @@ const DraggableList: React.FC<DraggableListProps> = ({ categories, onRightIconCl
             items: source_items,
         };
         setCategoryList(new_category_list);
+        onDrag && onDrag(new_category_list);
+    };
+
+    const handleOnDragStart = (start: any) => {
```

Comment:
start: should be some sort of object instead of any. Cant tell the type should be without seeing onDragStart in the Context

Reply:
I'll check, the type for this is coming from the library I used

Reply:
Parameters<React.ComponentProps<typeof DragDropContext>[onDragStart]>[0]

Reply:
Will it work?